google/google-how-to/gce/set-hostname-cloudinit.rst

```diff
@@ -0,0 +1,55 @@
+Set hostname with cloud-init on GCE
+===================================
+
+Hostnames can be set in many ways, including a user manually configuring the hostname, DHCP, and using a set-up tool like cloud-init.  On Ubuntu, most of these methods use the underlying ``hostname`` command, which sets the hostname statically if the ``/etc/hostname`` file exists and transiently if it does not.
```

Comment:
Since we are talking about static and dynamic hostnames and **D**HCP, can we use 'dynamically' instead of 'transiently everywhere in this page? It just seems to be the more common usage and hence easier to understand.

Reply:
Static and transient are the terms used by `hostnamectl` the systemd service that usually underpins `hostname` and which is what is affected by the existence or nonexistence of `/etc/hostname`.  The dynamically set hostname (e.g. the hostname coming from DHCP) will come in regardless.  Basically: transient hostname < dynamically set hostname < statically set hostname.